main/system.h

```diff
@@ -55,21 +48,22 @@
 #endif
 
 #if defined(IS_ATHENA)
-#define CSP_SCI sciREG2
-#define ADCS_SCI sciREG4
-#define DFGM_SCI sciREG1
-#define UHF_SCI CSP_SCI
+    #define CSP_SCI sciREG2  //UART2
+    #define ADCS_SCI sciREG3 //UART4
+    #define DFGM_SCI sciREG4 //UART1
+    #define UHF_SCI CSP_SCI  //UART2
 #if defined(IS_3U)
-#define GPS_SCI sciREG3
+    #define GPS_SCI sciREG1  //UART3
 #else if defined(IS_2U)
-#define PAYLOAD_SCI sciREG3
+    #define PAYLOAD_SCI sciREG1  //UART3
 #endif
 #else // These values are expected to be adjusted based on what the developer is working on
-#define CSP_SCI sciREG4
-#define GPS_SCI sciREG2
-#define ADCS_SCI sciREG3
-#define PAYLOAD_SCI 0
-#define DFGM_SCI 0
+    #define CSP_SCI sciREG2
+    #define GPS_SCI sciREG1
+    #define ADCS_SCI sciREG3
+    #define PAYLOAD_SCI sciREG1
```

Comment:
having multiple definitions of the same sciREG will cause issues in the interrupt switch() statements. We need to come up with a way to exclude compiling certain things based on the payload installed. Probably similar to IS_ATHENA